server/http.go

```diff
@@ -59,67 +78,102 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 }
 
 func (p *Plugin) serveHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) (int, error) {
-	switch r.URL.Path {
+	var err error
+	path := r.URL.Path
+	instanceURL := ""
+	instanceURL, path = splitInstancePath(path)
+	callbackInstanceID := types.ID("")
+
+	// Add any "callback" URLs here so that the caller instance is properly identified
```

Comment:
Need a better comment: that _some_ routes expect the instance ID prefix.

Reply:
I'm trying to rethink the wording here, but I think this comment makes enough sense as is.